src/graph_compute/common.h

```diff
@@ -75,17 +92,6 @@
   #define cilk_sync
 #endif
 
-typedef uint64_t vid_t;  // vertex id type
-
-struct vertex_t {
-  vid_t id;
-  vid_t priority;
-  vid_t dependencies;
-  volatile vid_t satisfied;
-  vid_t cntEdges;
-  vid_t * edges;
-  double data;
-};
-typedef struct vertex_t vertex_t;
+//  typedef uint32_t vid_t;  // vertex id type
```

Comment:
I moved the vertex_t definition to update_function.h, in order to facilitate it's composition of three pieces, as I described in the recent email:
a sched_t which is defined by each scheduling algorithm
a data_t which is defined by each application (e.g., pagerank and mass-spring-dashpot)
and vid_t * edges, vid_t cntEdges, used by everyone

The vid_t definition is in libgraphio.h